packages/swing-store/test/test-exportImport.js

```diff
@@ -225,6 +239,8 @@ async function testExportImport(
 
   t.deepEqual(exportData, feedData);
   t.deepEqual(exportData, [
+    [`bundle.${bundleID1}`, `${bundleID1}`],
```

Comment:
here's the part that needs changing, if you don't go for isolated minimal packages to hold the `faux-module.js` files